src/Microsoft.VisualStudio.Editors/PropPages/ApplicationPropPage.vb

```diff
@@ -676,6 +676,8 @@ Namespace Microsoft.VisualStudio.Editors.PropertyPages
 
             'Populate the target framework combobox
             PopulateTargetFrameworkComboBox(Me.TargetFramework)
+
+            Me.AssemblyInfoButton.Visible = Not Utils.IsTargetingDotNetCore(Me.ProjectHierarchy)
```

Comment:
Capabilties are not a CPS-only concept. There's a hierarchy property you can query to get the project node's capabilities I think.